app/javascript/packs/user_profile/body.jsx

```diff
@@ -123,9 +125,9 @@ class Body extends React.Component {
             <div className="col-md-5">
               <h2>Info</h2>
               <h3>
-                {firstName}
+                {firstName || (<span style={{color: "grey", float: "left"}}>No first name</span>)}
```

Comment:
Space after second '{' and before first '}', singleqoutes for strings
`{firstName || (<span style={{ color: 'grey', float: 'left' }}>No first name</span>)}`